data/859/017/79/85901779.geojson

```diff
@@ -4,6 +4,8 @@
   "properties": {
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
+    "edtf:deprecated":"12-21-2020",
+    "mz:is_current":0,
```

Comment:
"Cringasi" looks like a legitimate neighbourhood.. are we sure this neighbourhood should be removed / marked as not current?

Reply:
This is actually "Fundu Crîngași" - https://spelunker.whosonfirst.org/id/85901779/ which does not exist, and its polygon covers large areas from the city centre.